src/utils.ts

```diff
@@ -0,0 +1,147 @@
+// eslint-disable-next-line max-classes-per-file
+import { IAFile, IAMD, TimelineMoment } from './interfaces';
+
+export class Moment implements TimelineMoment {
+  altText: string;
+
+  desktopImg: string;
+
+  mobileImg: string;
+
+  link: string | undefined;
+
+  constructor() {
+    this.altText = '';
+    this.desktopImg = '';
+    this.mobileImg = '';
+    this.link = undefined;
+  }
+}
```

Comment:
Since this class isn't actually doing anything with the data, it's just storing it, you don't actually need it and can just use the `TimelineMoment` interface directly. For instance, wherever you use `const moment = new Moment()`, you can just do this:
```
const moment: TimelineMoment = {}
moment.altText = 'foo';
moment.desktopImg = 'bar'
```
or
```
const moment: TimelineMoment = {
  altText: 'foo',
  desktopImg: 'bar'
}
```

Reply:
using the latter